libc-test/build.rs

```diff
@@ -733,6 +733,7 @@ fn test_solarish(target: &str) {
     cfg.define("_XOPEN_SOURCE", Some("700"));
     cfg.define("__EXTENSIONS__", None);
     cfg.define("_LCONV_C99", None);
+    cfg.define("_STRUCTURED_PROC", Some("1"));
```

Comment:
For what did you need this define?

Reply:
to enable new struct instead (see sys/procfs.h and sys/old_procfs.h)

Reply:
If you use the plain `procfs.h` include as noted below, that will be defined for you.

Reply:
Right, `procfs.h` is the public header we should be using for this.